packages/datatrak-web/src/layout/AuthPageLayout.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { Outlet } from 'react-router';
+import { Background as BaseBackground } from './BackgroundPageLayout';
+import { PageContainer } from '../components';
+
+export const Background = styled(BaseBackground)`
+  h2 {
+    font-weight: ${({ theme }) => theme.typography.fontWeightBold};
+  }
+  h3 {
+    margin-top: 0.32rem;
+  }
+  .MuiFormControl-root {
```

Comment:
I feel like these tweaks to the styles might belong in the ui-components forms. That would mean that they will also apply to tupaia-web but I think the plan is to keep datatrak and tupaia-web the same. Can you please check with Flic?